index.js

```diff
@@ -146,6 +154,30 @@ class Client extends EventEmitter {
     })
   }
 
+  get transport () {
+    return this[kTransport]
```

Comment:
it can be as simple as `_transport` instead of using `Symbol`. not 100% safe, but Symbol doesn't provide full safety either.